test/exercise/fp/solution.rb

```diff
@@ -4,12 +4,34 @@ class << self
       # Обратиться к параметрам фильма можно так:
       # film["name"], film["rating_kinopoisk"], film["rating_imdb"],
       # film["genres"], film["year"], film["access_level"], film["country"]
-      def rating(_array)
-        0
+      def rating(array)
+        filtered_films =
+          array
+          .select { |film| film['rating_kinopoisk'].to_f.positive? }
+          .select { |film| film['country'] }
+          .select { |film| film['country'].split(',').length > 1 }
```

Comment:
it's can be made in a one check
+ country / rating can be null, need to handle it